source/features/prevent-pr-merge-panel-opening.tsx

```diff
@@ -17,17 +17,9 @@ function init(signal: AbortSignal): void {
 }
 
 void features.add(import.meta.url, {
-	asLongAs: [
-		// The user is a maintainer, so they can probably merge the PR
-		() => select.exists('.discussion-sidebar-item .octicon-lock'),
-	],
 	include: [
 		pageDetect.isPRConversation,
 	],
-	exclude: [
-		() => select.exists('#partial-discussion-header [title="Status: Draft"]'),
-	],
```

Comment:
It doesn't work with `awaitDomReady: false` and it's not needed with the observer